hs/Makefile

```diff
@@ -10,11 +10,30 @@ install-mo:
 mo:
 	@which mo 1>/dev/null || (echo 'FAILURE: Please call `make install-mo` first' && exit 1)
 
-package.yaml: package.mo.yaml ../VERSION mo
+package.yaml: package.mo.yaml ../VERSION
```

Comment:
Why the change on this line?

Reply:
Because `mo` is phony, it thinks that this always needs to be rebuilt